cluster/node-pools/master-default/userdata.yaml

```diff
@@ -246,11 +246,28 @@ write_files:
               memory: 50Mi
           args:
             - --tokens-file=/etc/kubernetes/config/tokenfile.csv
+
+            # Collaborator roles
+            - --role-mapping=Administrator=cn=Administrator,ou=collaborators,ou=Kubernetes,ou=apps,dc=zalando,dc=net
+            - --role-mapping=ReadOnly=cn=ReadOnly,ou=collaborators,ou=Kubernetes,ou=apps,dc=zalando,dc=net
+{{- if eq .Cluster.Environment "test"}}
+            - --role-mapping=PowerUser=cn=Deployer,ou=collaborators,ou=Kubernetes,ou=apps,dc=zalando,dc=net
+{{- else if eq .Cluster.Environment "e2e"}}
+            - --role-mapping=Administrator=cn=Contributor,ou=collaborators,ou=Kubernetes,ou=apps,dc=zalando,dc=net
+{{- end}}
+
+            # Member roles
+{{- if eq .Cluster.Environment "production"}}
+            - --role-mapping=Manual=cn=Manual,ou={{.Cluster.InfrastructureAccount}}:{{.Cluster.Region}}:{{.Cluster.LocalID}},ou=Kubernetes,ou=apps,dc=zalando,dc=net
```

Comment:
nit: `{{.Cluster.InfrastructureAccount}}:{{.Cluster.Region}}:{{.Cluster.LocalID}}` could just be `{{.Cluster.ID}}` right? :)

Reply:
Good point.